bin/test/i18n/resources/pegasus/houtofcode/test_sync_out.rb

```diff
@@ -2,53 +2,30 @@
 require_relative '../../../../../i18n/resources/pegasus/hourofcode/sync_out'
 
 describe I18n::Resources::Pegasus::HourOfCode::SyncOut do
-  def around
-    FakeFS.with_fresh {yield}
+  let(:described_class) {I18n::Resources::Pegasus::HourOfCode::SyncOut}
+  let(:described_instance) {described_class.new}
+
+  let(:crowdin_locale) {'expected_crowdin_locale'}
+  let(:i18n_locale) {'expected_i18n_locale'}
```

Comment:
Not a blocker, but we should be consistent on the naming of testing locales, some older test files use `te-ST` as a non-English locale.